Website/Website/App_Code/DAO/VisitorDAO.cs

```diff
@@ -111,22 +108,37 @@ public Visitor LoginVisitor(string email)
             return null;
```

Comment:
Als er iets mis gaat return je null en is de kans op een nullpoint exception bizar groot. Je kunt dat beter een leeg object returnen

Reply:
in dit geval krijg je geen null pointer exception omdat "null" in principe in een session gegooid kan worden, maar ik denk wel dat het beter is als er idd een lege visitor object teruggestuurd wordt